website/src/models/group.js

```diff
@@ -210,23 +210,16 @@ schema.methods.startQuest = async function startQuest (user) {
     user.markModified('party.quest');
   }
 
-  // Remove the quest from the quest leader items (if he's the current user)
+  // Remove the quest from the quest leader items (if they are the current user)
```

Comment:
Yeah, I know that's probably not as performative, but I think it's worth it for code clarity and reducing duplication. Plus it removes the need to pop the quest leader of the array, and then put them back on after the second update call so that they can receive the quest started email. 

Do you have strong feelings about it?